src/core/structured-log.js

```diff
@@ -277,12 +277,15 @@
       invoke('TRACE', mt, arguments);
     };
 
-    self.information = function(messageTemplate) {
+    self.verbose = self.trace;
```

Comment:
That sounds like a valid alternative too - may still be best finalised in a separate ticket if that works for you.